README.md

```diff
@@ -35,6 +35,8 @@ This is in the `slurm-dev-environment` directory: see [slurm-dev-environment/REA
 
 There is also an apache web server, as a mock "transfer services area" for serving data files over HTTP.
 
+There is also an openresty/nginx server with the http_secure_link_module, as a "private-data-server" for serving private data files with secure pre-signed URLs. This is used for private studies and analyses.
```

Comment:
First time I head about openresty.. seems like a healthy project and BSD licensed.

Reply:
Yeh, I wasn't aware until I started looking for ready-to-use images with the nice plugins and/or lua-for-nginx support. Good stuff.